packages/angular/projects/ui-angular/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: '../../.eslintrc.js',
+};
```

Comment:
This is the sub-project lint settings for `@aws-amplify/ui-react`.

Doesn't do much, but left it here to avoid making changes to config structure that `ng-packagr` created.